data/curated/wa-cannabis/cleaning.R

```diff
@@ -0,0 +1,10 @@
+# data/curated/wa-cannabis/cleaning.R
+
+library(readr)
+library(tidyverse)
+library(lubridate)
+
+# Load dataset from your GitHub raw link
+dataset_url <- "https://raw.githubusercontent.com/Ensign-Analytics/tidytuesday-wa-cannabis/main/wa_cannabis_sample.csv"
```

Comment:
This is meant to be a reproducible script, but this URL does not exist, or perhaps is in a private repository.